py/py_sparkling/ml/algos.py

```diff
@@ -132,7 +150,10 @@ def __init__(self, featuresCols=[], labelCol="label", allStringColumnsToCategori
                  stoppingTolerance=0.001, stoppingMetric="AUTO", nfolds=5, convertUnknownCategoricalLevelsToNa=True, seed=-1,
                  sortMetric="AUTO", balanceClasses=False, classSamplingFactors=None, maxAfterBalanceSize=5.0,
                  keepCrossValidationPredictions=True, keepCrossValidationModels=True, maxModels=0,
-                 predictionCol="prediction", **deprecatedArgs):
+                 predictionCol="prediction",
```

Comment:
nit: Shouldn't the new parameter use the same formatting as previous parameters?

Reply:
I found myself really hard to read the options as they are now, wanted to switch to different formatting, one option per line, but would like to do it when we have more time

Reply:
ok

Reply:
But if you prefer the consistency, I can switch it back, no worries :)

Reply:
I would prefer to be consistent and change formatting of all parameters in a different PR. But this a really minor thing. If you leave it as it's, it works for me as well.

Reply:
No worries, valid point, will keep the current formatting :)

Reply:
Fixed, thx